docs/kratos/social-signin/35_gitlab.mdx

```diff
@@ -4,8 +4,76 @@ title: Add GitLab as a social sign-in provider in Ory
 sidebar_label: GitLab
 ---
 
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+
 # GitLab
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add GitLab as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the GitLab logo to start the configuration.
+3. Copy the Redirect URI and save it for later use.
+4. [Create a GitLab OAuth2 Application](https://docs.gitlab.com/ee/integration/oauth_provider.html#adding-an-application-through-the-profile),
+  noting the following points:
+    - Paste the saved **Redirect URI** from Ory into the **Redirect URI** field in the GitLab form.
+    - Enable the relevant **Scopes** for the application, by ticking the relevant boxes in the GitLab form:
+        - `read_user`
+        - `openid`
+        - `profile`
+        - `email`
+5. After clicking **Save application** on the GitLab form, you see a summary of the registered application's properties.
+    :::caution
+    Do not click away from this page before copying the **Secret** field.
+    This is the only opportunity you will get to copy the secret.
+    :::
```

Comment:
There needs to be an empty line before and after the note. 
Also: use contractions, 0 conditional, "click away" isn't unambiguous (if I switch the tab will the secret remain visible?). There should be a better way to phrase this.

Reply:
Ah, I was a bit confused by this, because Docusaurus rendered the content perfectly without the empty lines. But if our convention is to include empty lines, I will go ahead and add them.

Reply:
Added the empty lines and rephrased the "caution" admonition.